<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#E5EEF8;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leftLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #487DFC;
border-top-right-radius: 70px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Condensed</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Skin Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
}

QPushButton:hover {
    color: #00DD63;
	border: 1px solid  #00DD63;
	border-top-right-radius:10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="C:/Users/Byke/AppData/Local/Programs/Python/Python310/Lib/site-packages/qt6_applications/Qt/bin/setting_frame_res.qrc">
      <normaloff>:/Icons/D:/Projects/newPy/QT_Project/Player/res/Icons/arrow-left.svg</normaloff>:/Icons/D:/Projects/newPy/QT_Project/Player/res/Icons/arrow-left.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
}

QPushButton:hover {
    color: #00DD63;
	border: 1px solid red;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="C:/Users/Byke/AppData/Local/Programs/Python/Python310/Lib/site-packages/qt6_applications/Qt/bin/setting_frame_res.qrc">
      <normaloff>:/Icons/D:/Projects/newPy/QT_Project/Player/res/Icons/x.svg</normaloff>:/Icons/D:/Projects/newPy/QT_Project/Player/res/Icons/x.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
}

QPushButton:hover {
    border: 1px solid  #00DD63;
	border-top-left-radius:10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="C:/Users/Byke/AppData/Local/Programs/Python/Python310/Lib/site-packages/qt6_applications/Qt/bin/setting_frame_res.qrc">
      <normaloff>:/Icons/D:/Projects/newPy/QT_Project/Player/res/Icons/minus.svg</normaloff>:/Icons/D:/Projects/newPy/QT_Project/Player/res/Icons/minus.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="C:/Users/Byke/AppData/Local/Programs/Python/Python310/Lib/site-packages/qt6_applications/Qt/bin/setting_frame_res.qrc"/>
 </resources>
 <connections/>
</ui>
